scheds/rust/meson.build

```diff
@@ -1,5 +1,23 @@
+# the previous scheduler in the compile sequence
+sched = []
+
+# to add a new scheduler, assign the output of your custom_target to sched
+# and add sched as a dependency to your custom_target. For example:
```

Comment:
Would you mind adding a bit of explanation on why we're doing this?

Reply:
Fixed. Let me know if the explanation is not detailed enough.